src/window/window.js

```diff
@@ -13,13 +13,19 @@ const windowMapping = {};
  * Open a given window; if it is already open, focus it.
  * @param {string} name The window identifier; this controls window re-use.
  * @param {string} url The URL to load into the window.
+ * @param {Boolean} restoreMinimizedWindow If it's minimized, restore it.
+ *    This is for when the user tries to run a second instance, and that instance
+ *    focuses this one and shuts itself down.
  * @param {Electron.WebPreferences} prefs Options to control the new window.
  */
-function createWindow(name, url, prefs) {
+function createWindow(name, url, restoreMinimizedWindow, prefs, restoreWindow=false) {
```

Comment:
That last argument doesn't seem to be documented.  Or even used.
```suggestion
function createWindow(name, url, restoreMinimizedWindow, prefs) {
```

Actually, I don't understand why it exists at all: we only ever call this when we intend for the window to be in the user's face (initial startup, or they invoked a menu entry to show the window). When would we want to _not_ restore the window from minimized?